store/tikv/mock-tikv/pd.go

```diff
@@ -70,6 +70,11 @@ func (c *pdClient) GetRegionByID(ctx context.Context, regionID uint64) (*metapb.
 }
 
 func (c *pdClient) GetStore(ctx context.Context, storeID uint64) (*metapb.Store, error) {
+	select {
+	case <-ctx.Done():
+		return nil, ctx.Err()
+	default:
+	}
```

Comment:
Why not put this logic to `loadStoreAddr` directly?
That needn't check error cause.